_includes/links.md

```diff
@@ -29,7 +29,8 @@
 [rubygems]: https://rubygems.org/pages/download/
 [styles]: https://github.com/swcarpentry/styles/
 [training]: http://swcarpentry.github.io/instructor-training/
-[workshop-repo]: {{ site.workshop_repo }}
+[workshop-repo]: https://github.com/swcarpentry/workshop-template
```

Comment:
This line is correct. Please see https://github.com/swcarpentry/lesson-example/blob/gh-pages/_config.yml#L28.